changelog/12652.txt

```diff
@@ -0,0 +1,4 @@
+```release-note:bug
+Add missing read unlock calls in transit backend code.
```

Comment:
I don't have experience writing changelog entries but it seems to me that as the audience are Vault users/operators, you don't need to mention detailed information such as methods called etc. Perhaps something more generic along the lines of: fixed a problem with the transit cache, and made it unnecessary to reload the backend when the configuration changes.

Reply:
We probably already have the changelog entry for the overall cache issue, so I think we can just remove this.

Reply:
Thank you. Removed the changelog entry for this